libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationView.kt

```diff
@@ -168,6 +173,36 @@ class NavigationView @JvmOverloads constructor(
      */
     val api: NavigationViewApi = MapboxNavigationViewApi(navigationContext.store)
 
+    /**
+     * A flow with an object to play voice instructions.
+     */
+    val voiceInstructionsPlayerFlow = audioGuidance
+        .voiceInstructionsPlayerFlow
+        .distinctUntilChanged()
+
+    /**
+     * Register [VoiceInstructionsPlayerObserver]
+     */
+    fun registerVoiceInstructionsPlayerObserver(observer: VoiceInstructionsPlayerObserver) {
+        audioGuidance.registerVoiceInstructionsPlayerObserver(observer)
+    }
+
+    /**
+     * Unregister [VoiceInstructionsPlayerObserver]
+     */
+    fun unregisterVoiceInstructionsPlayerObserver(observer: VoiceInstructionsPlayerObserver) {
+        audioGuidance.unregisterVoiceInstructionsPlayerObserver(observer)
+    }
+
+    /**
+     * Unregister all [VoiceInstructionsPlayerObserver]
+     *
+     * @see [registerVoiceInstructionsPlayerObserver]
+     */
+    fun unregisterAllVoiceInstructionsPlayerObservers() {
+        audioGuidance.unregisterAllVoiceInstructionsPlayerObservers()
+    }
+
```

Comment:
Let's move all the changes in this file to `NavigationViewApi`. We are avoiding adding new APIs directly to `NavigationView` unless absolutely necessary.

Reply:
should we save 2 API's?
- flow `voiceInstructionsPlayerFlow` for kotlin users
- observer (register/unregister/unregisterAll) for java/kotlin users
or save observer only?